packages/performance/src/resources/trace.test.ts

```diff
@@ -116,10 +116,14 @@ describe('Firebase Performance > trace', () => {
 
       expect(trace.getMetric('cacheHits')).to.eql(600);
     });
+
+    it("throws error if metric doesn't exist and has invalid name", () => {
+      expect(() => trace.incrementMetric('_invalidMetric', 1)).to.throw();
+    });
   });
 
   describe('#putMetric', () => {
-    it('creates new metric if one doesnt exist.', () => {
+    it('creates new metric if one doesnt exist and has valid name.', () => {
```

Comment:
"doesn't" instead of "doesnt"

Reply:
done

Reply:
Unfortunately the code formatter automatically changes strings to use double quotes when they contain an escaped single quote. I discussed with alikn and we decided to continue using single quotes for consistency and to accept the grammatical tradeoff.